<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comunicaci√≥n entre PC - Mbed</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 59, 60);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>18</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 66, 33);
color: rgb(255, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>   PUNTAJE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>ESTADO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(52, 255, 93);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>ACIERTOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 11, 11);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string> ERRORES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>  FALLOS</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(92, 255, 74);
</string>
    </property>
    <property name="text">
     <string>CONECTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
</string>
    </property>
    <property name="text">
     <string>DESCONECTAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(46, 59, 60);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>139</y>
      <width>961</width>
      <height>371</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>124</y>
      <width>203</width>
      <height>124</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>1</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuComunicaci_n">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Comunicaci√≥n</string>
     </property>
     <addaction name="actionEscaneo_de_Puertos"/>
     <addaction name="actionDesconectar"/>
     <addaction name="actionConectar"/>
    </widget>
    <addaction name="menuComunicaci_n"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionEscaneo_de_Puertos">
   <property name="text">
    <string>Escaneo de Puertos</string>
   </property>
  </action>
  <action name="actionDesconectar">
   <property name="text">
    <string>Desconectar</string>
   </property>
   <property name="toolTip">
    <string>Desconectar</string>
   </property>
  </action>
  <action name="actionConectar">
   <property name="text">
    <string>Conectar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
